lasagne/utils.py

```diff
@@ -244,11 +243,13 @@ def create_param(spec, shape, name=None):
 
     Notes
     -----
-    This method should be used in the constructor when creating a
-    :class:`Layer` subclass that has parameters. This enables the layer to
+    This method is called by :meth:`Layer.add_param()` in the constructor
```

Comment:
Maybe we should change 'method' to 'function' as well, since technically a method is part of a class.